dash/pages.py

```diff
@@ -1016,6 +1016,14 @@ def create_CV_scores_table(series_data_dict):
 
             return CV_score_df
 
+        # Format to clean string so tables don't have very large numbers. anything larger than 4 characters can go to scientific notation.
```

Comment:
Since this converts the data into strings does this break the built in sorting?

Reply:
The `create_CV_scores_table` makes the table and does the sort first. 
Then it appears the rounding is done after the table is sort by MSE. 
By if it want's to be sorted a second time, the it will sort a 'str' and not by 'int'. 
```
            dataframe = create_CV_scores_table(series_data_dict)
            rounded_dataframe = dataframe.copy()
            # Round and format so that trailing zeros still appear
            for col in rounded_dataframe.columns:
                rounded_dataframe[col] = rounded_dataframe[col].apply(
                    cv_table_clean_notation
                )
```

Reply:
Yeah that's what I mean. Can you make sure that it doesn't break sorting client/browser side on the DataTable?

![image](https://user-images.githubusercontent.com/3364754/172522426-ab993a00-b3b2-4aad-bea0-fa8d5e94edc7.png)